decode.go

```diff
@@ -33,6 +34,24 @@ var (
 	}
 )
 
+type Bbox struct {
```

Comment:
Would this be better as `BBox` which better matches the `osmformat.proto` file of `HeaderBBox` or maybe even `BoundingBox` which is more descriptive?